README.md

```diff
@@ -40,6 +41,9 @@ $sitemap->addItem('http://example.com/mylink1');
 $sitemap->addItem('http://example.com/mylink2', time());
 $sitemap->addItem('http://example.com/mylink3', time(), Sitemap::HOURLY);
 $sitemap->addItem('http://example.com/mylink4', time(), Sitemap::DAILY, 0.3);
+// add some URLs of alternates links
+$sitemap->addAlternates('http://example.com/%s/mylink5', array('it','fr','de'));
```

Comment:
Alternate URLs aren't necessarily constructed like that. For example, it's common to have title in URL so for different languages titles may vary.

Reply:
You're right! I'm going to modify it to use several ways to create them. Thank you